tasks/e2e/cypress/integration/01-tutorial/codemods/Step0_2_GraphQL.js

```diff
@@ -7,13 +7,67 @@ import {
 
 import schemas from 'src/graphql/**/*.{js,ts}'
 import { db } from 'src/lib/db'
+import { logger } from 'src/lib/logger'
 import services from 'src/services/**/*.{js,ts}'
 
+const logTimingTraces = {
```

Comment:
Guessing you're two steps ahead of me here. But will this:
- be visible/available during CI runs?
- possible to collect and aggregate this from CI runs, e.g. with Datadog? Might be nice to be able to benchmark and watch for trends over time

Reply:
So, in test, logs should be silent. But I want to move the plugin out of the e2e test and into the framework instead -- and enable it via an option and rename it to useRedwoodLogger to match the api of the H/E handler.

Then in the test apps, we can simply say to use the logging and enable tracing.